tests/e2e/config/wordpress-debug-log/index.js

```diff
@@ -32,7 +32,7 @@ import { printReceived } from 'jest-matcher-utils';
  */
 import { logIgnoreList } from './log-ignore-list';
 
-const docker = new Docker( { socketPath: '/var/run/docker.sock' } );
```

Comment:
I was originally thinking we would add the above connectivity test to this file but it became unexpectedly complicated due to that file running in a Jest context (which messes with console output and exit calls) and being loaded before every test file. It only needs to be checked once before E2E runs.